Monika After Story/game/script-greetings.rpy

```diff
@@ -773,6 +773,30 @@ label greeting_latin:
     m 3eua "If you're wondering about what I said, it's simply 'We meet again! How are you?'"
     return
 
+#TODO needs additional dialogue so can be used for all aff
+init 5 python:
+    addEvent(
+        Event(
+            persistent.greeting_database,
+            eventlabel="greeting_esperanto",
+            unlocked=True,
+            aff_range=(mas_aff.NORMAL, None),
+        ),
+        code="GRE"
+)
+
+label greeting_esperanto:
+#    m "Mia kusenveturilo estas plena je angiloj."  # execute obligatory monty.py
+    m 1eub "Saluton, mia kara [player]."
+    m 5eub "Kiel vi fartas?"
+    m 2efb "Ĉu estas vi preta por la tago?"
```

Comment:
```suggestion
    m 3eub "Ĉu vi legas por kapti la tagon?"
```
According to google translate, the old line was `are you ready for the day`

Reply:
I wasn't able to give a response on the changes in time, but the reason for the difference is because I wasn't sure how best to translate the expression. On further recall, because the idiom exists in the original Latin, it should work, but I had opted for a less idiomatic phrase.

However, the verb `legas` means `to read (present tense)`, so this line now says `"Are you reading to seize the day?"`

To stick to the idiom, it should be
```suggestion
    m 3eub "Ĉu vi pretas por kapti la tagon?"
```

Reply:
@MisterSimple add this to #4927 please